nosqlbench/http-jsonapi-crud-basic.yaml

```diff
@@ -1,4 +1,4 @@
-min_version: "5.17.1"
+min_version: "5.17.3"
```

Comment:
trying to run this locally, produces strange results.. the scenario ends correctly and no errors are reported, but on the server-side metrics I am seeing a lot of client errors:

```
# HELP http_server_requests_seconds  
# TYPE http_server_requests_seconds summary
http_server_requests_seconds_count{method="POST",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/v1/{namespace}/{collection}",} 40000.0
http_server_requests_seconds_sum{method="POST",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/v1/{namespace}/{collection}",} 620.394043765
http_server_requests_seconds_count{method="POST",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/v1",} 2.0
http_server_requests_seconds_sum{method="POST",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/v1",} 1.928188204
http_server_requests_seconds_count{method="POST",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/v1/{namespace}",} 4.0
http_server_requests_seconds_sum{method="POST",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/v1/{namespace}",} 14.483613413
http_server_requests_seconds_count{method="POST",module="sgv2-jsonapi",outcome="CLIENT_ERROR",status="RESET",uri="/v1/{namespace}/{collection}",} 40000.0
http_server_requests_seconds_sum{method="POST",module="sgv2-jsonapi",outcome="CLIENT_ERROR",status="RESET",uri="/v1/{namespace}/{collection}",} 617.666413847
http_server_requests_seconds_count{method="POST",module="sgv2-jsonapi",outcome="CLIENT_ERROR",status="RESET",uri="/v1/{namespace}",} 4.0
http_server_requests_seconds_sum{method="POST",module="sgv2-jsonapi",outcome="CLIENT_ERROR",status="RESET",uri="/v1/{namespace}",} 12.24249949
http_server_requests_seconds_count{method="GET",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/favicon.ico",} 1.0
http_server_requests_seconds_sum{method="GET",module="sgv2-jsonapi",outcome="SUCCESS",status="200",uri="/favicon.ico",} 0.012527204
http_server_requests_seconds_count{method="POST",module="sgv2-jsonapi",outcome="CLIENT_ERROR",status="RESET",uri="/v1",} 2.0
http_server_requests_seconds_sum{method="POST",module="sgv2-jsonapi",outcome="CLIENT_ERROR",status="RESET",uri="/v1",} 1.926456839
```

Reply:
Created this: https://github.com/stargate/jsonapi/issues/411

Reply:
Thanks, we can continue investigation there.